docker-compose/docker-compose.yaml

```diff
@@ -87,6 +87,7 @@ services:
       - 'PRECISE_CODE_INTEL_API_SERVER_URL=http://precise-code-intel-api-server:3186'
       - 'GRAFANA_SERVER_URL=http://grafana:3370'
       - 'GITHUB_BASE_URL=http://github-proxy:3180'
+      - 'PROMETHEUS_SERVER_URL=http://prometheus:9090'
```

Comment:
Whoops, sorry, I forgot to mention you should also set this below under the `sourcegraph-frontend-internal:` section too for consistency sake (so the two remain in-sync). Can you update for that?